android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java

```diff
@@ -312,6 +331,16 @@ public TiScrollViewLayout getLayout()
 
 		@Override
 		public boolean onTouchEvent(MotionEvent event) {
+			if (event.getAction() == MotionEvent.ACTION_MOVE && !isTouching) {
```

Comment:
Should probably put these conditions below:
```Java
if (event.getAction() == MotionEvent.ACTION_MOVE && !mScrollingEnabled) {
```
to not fire when `scrollingEnabled` is false?